src/main/java/hillel/spring/repository/HelloRepository.java

```diff
@@ -0,0 +1,19 @@
+package hillel.spring.repository;
+
+import hillel.spring.database.Database;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class HelloRepository {
+
+    private final Database database;
+
+    public HelloRepository() {
+        database = new Database();
```

Comment:
Looks like a `Database` should be a bean as well (and should be injected).

Reply:
@ Bean and @Autowired?